scripts/managed-services/get-kafka-instance-state/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+file: get-kafka-instance-state.sh
+name: managed-services/get-kafka-instance-state
+description: |
+  Print out the logs for the specified Kafka instance.
+  The following commands will create a directory with all instance information:
+
+  ocm backplane managedjob logs <job-name> > mykafkadebug.tgz
+  tar -xvf mykafkadebug.tgz
+author: racheljpg
+allowedGroups:
+  - SREP
+  - CSSRE
+  - CEE
+rbac:
+  roles:
+    - namespace: "redhat-kas-fleetshard-operator"
```

Comment:
This doesn't look right.  This script is nothing to do with the fleetshard operator, so it doesn't seem right to use that operator's namespace.  Also there a practical problem, the name of this namespace differs between stage and production redhat-kas-fleetshard-operator and redhat-kas-fleetshard-operator-qe.

Can we use the default namespace?

@StevenTobin could I get your opinion on this please?

Reply:
Yeah @racheljpg and I discussed this, ideally it would run in the namespace of the kafka cluster you are targeting but failing that I recommended a namespace that we create but as you rightly point out the namespace difference between prod and stage is a problem for that.

I don't think we can specify the kafka namespace since that is specified as a variable but I would love confirmation on that.

I would be fine with it running in the default namespace.

Reply:
> I don't think we can specify the kafka namespace since that is specified as a variable but I would love confirmation on that.

For that to work, the rbac rules would have to support parameterization.  I wouldn't put money on ocm backplane supporting that.